src/Components/Server/src/Circuits/CircuitRegistry.cs

```diff
@@ -297,6 +296,31 @@ private void DisposeTokenSource(DisconnectedCircuitEntry entry)
             }
         }
 
+        internal ValueTask Terminate(string circuitId)
```

Comment:
nit: public. This should probably be `TerminateAsync` for consistency with other things.